### ISSUE TITLE
Two JSON libs

### ISSUE THREAD

**Issue by fridobox** [bywatersolutions/koha-plugin-batch-permissions-modifier#10]
https://github.com/bywatersolutions/koha-plugin-batch-permissions-modifier/blob/cc90cf58658a267bae7a02ff49895a6462b1629a/Koha/Plugin/Com/ByWaterSolutions/BatchPermissionsModifier.pm#L6

I see this error in logs :
```
Par contre au lancement du script :
Prototype mismatch: sub Koha::Plugin::Com::ByWaterSolutions::BatchPermissionsModifier::decode_json ($) vs none at /usr/share/perl/5.30/Exporter.pm line 66.
 at /home/koha/lib/koha-plugin-batch-permissions-modifier/Koha/Plugin/Com/ByWaterSolutions/BatchPermissionsModifier.pm line 7.
```

Both the JSON and Mojo::JSON libs have a method "decode_json".
Is the use JSON needed ?

**Comment by fridobox** [comment]
Any news ?

**Comment by fridobox** [comment]
Fixed by 693fc680fdd71741e20c5a48bf5c9e3e6a3fb8ce
